src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2908,11 +2908,12 @@ void SpellMgr::LoadSpellCustomAttr()
                                 if (spellInfo->Id != 69649 && spellInfo->Id != 71056 && spellInfo->Id != 71057 && spellInfo->Id != 71058 && spellInfo->Id != 73061 && spellInfo->Id != 73062 && spellInfo->Id != 73063 && spellInfo->Id != 73064) // Sindragosa Frost Breath
                                     if (spellInfo->SpellFamilyName != SPELLFAMILY_MAGE || !(spellInfo->SpellFamilyFlags[0] & 0x20)) // frostbolt
                                         if (spellInfo->Id != 55095) // frost fever
-                                            if (spellInfo->SpellFamilyName != SPELLFAMILY_WARLOCK || !(spellInfo->SpellFamilyFlags[1] & 0x40000)) // Haunt
-                                            {
-                                                spellInfo->AttributesCu |= SPELL_ATTR0_CU_BINARY_SPELL;
-                                                break;
-                                            }
+                                            if(spellInfo->Id != 62457 && spellInfo->Id != 65370) // Hodir Icicles
+                                                if (spellInfo->SpellFamilyName != SPELLFAMILY_WARLOCK || !(spellInfo->SpellFamilyFlags[1] & 0x40000)) // Haunt
+                                                {
+                                                    spellInfo->AttributesCu |= SPELL_ATTR0_CU_BINARY_SPELL;
+                                                    break;
+                                                }
```

Comment:
from the codestyle perspective I'd rather just add:

```
if (spellInfo->Id == 62457 || spellInfo->Id == 65370) // Hodir Icicles
{
    continue;
}
```

and so avoid to move `if (spellInfo->SpellFamilyName ....` further

Reply:
Good idea